### DISCUSSION TITLE
Transcoded RTMP Push

### DISCUSSION THREAD

**Discussion by irlkitcom** [OvenMediaLabs/OvenMediaEngine#1586] [Help]
Is it possible to transcode the incoming RTMP stream before pushing it to another RTMP destination?

**Comment by getroot** [comment]
Yes, you can select a track from the OutputStream and push it again.

https://airensoft.gitbook.io/ovenmediaengine/rest-api/v1/virtualhost/application/push

**Reply by irlkitcom** [reply]
If I have Origins that are only doing this and pushing, no delivering video directly, is there any advantage to using edge servers?
